examples/SDM_example.py

```diff
@@ -0,0 +1,204 @@
+# ANT+ - Stride Based Speed and Distance Sensor Example
+#
+# SDM demo working with OpenAnt Library (https://github.com/Tigge/openant)
+# For further details on Speed & Distance Sensor, check out the thisisant.com webpage
+
+
+from __future__ import absolute_import, print_function
+
+import array
+import threading
+import logging
+import time
+
+from ant.easy.node import Node
+from ant.easy.channel import Channel
+from ant.base.message import Message
+from ant.base.commons import format_list
+
+
+import struct
+import sys
+
+# Definition of Variables
+NETWORK_KEY= [0xb9, 0xa5, 0x21, 0xfb, 0xbd, 0x72, 0xc3, 0x45]
+Device_Type = 124  # 124 = Stride & Distance Sensor
+Device_Number = 12345  # Change if you need.	
+Channel_Period = 8134
+Channel_Frequency = 57	
+
+# Fictive Config of Treammill 
+TreadmillSpeed = 2.777  # m/s => 10km/h
+TreadmillCadence = 160
+		
+##########################################################################
+
+class AntSendDemo:
+
+    def __init__(self):
+
+        self.ANTMessageCount = 0
+        self.ANTMessagePayload = array.array('B', [0, 0, 0, 0, 0, 0, 0, 0])
+
+		# Init Variables, needed

```

Comment:
Comments throughout seems to have tabs instead of spaces. Don't mix tabs and spaces, use 2 spaces for indentation.

Reply:
done